tests/spec/core/data-include-spec.js

```diff
@@ -46,7 +46,7 @@ describe("Core — Data Include", () => {
     expect(missing).toBeNull();
     const included = doc.getElementById("replacement-test");
     expect(included).toBeTruthy();
-    const heading = doc.querySelector("#replacement-test > h3");
+    const heading = doc.querySelector("#replacement-test > div > h3");
```

Comment:
```suggestion
    const heading = doc.querySelector("#replacement-test > .header-wrapper > h3");
```

Just so we're clearer what this div is. (and ditto in other places)

Reply:
I’ll see if i can find replace. But not spending any more cycles on this. It’s a time suck.